src/Native/System.Native/pal_interfaceaddresses.cpp

```diff
@@ -44,8 +44,7 @@ extern "C" int32_t EnumerateInterfaceAddresses(IPv4AddressFound onIpv4Found,
         {
             // IP Address
             IpAddressInfo iai = {
-                .InterfaceIndex = interfaceIndex,
-                .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
+                .InterfaceIndex = interfaceIndex, .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
```

Comment:
Do we like how this is formatted now? I kind of prefer it on two lines, it's a little bit easier to read I think. Might just be a preference, though, and I don't have as much experience looking at C++ code :smile: